test/truster/README.md

```diff
@@ -0,0 +1,777 @@
+```javascript
+$ halmos --solver-timeout-assertion 0 --function check_truster -vvvvv
+Path #108:
+...
+Trace:
+            CALL 0xaaaa0003::flashLoan(...)
+            ...
+                CALL 0xaaaa0003::Extract(p_data_bytes_334a71c_07())(Extract(p_data_bytes_334a71c_07()))
+                ↩ CALLDATALOAD 0x (error: NotConcreteError('symbolic CALLDATALOAD offset: 4 + Extract(7391, 7136, p_data_bytes_334a71c_07)'))
+...
+```
+The error is some symbolic call in **TrusterLenderPool::flashLoan()** function:
+```solidity
+function flashLoan(uint256 amount, address borrower, address target, bytes calldata data)
+...
+{
+...
+    target.functionCall(data);
+}
+```
+Yes, since the **data** parameter also turned out to be symbolic, we got into the same pattern again and got the same error. However, this time, knowing that we can get to a symbolic call of a symbolic address in any contract, we will use a more universal solution!
+## Global storage
+Let's create a library global storage contract that can be accessed from anywhere:
+```solidity
+// SPDX-License-Identifier: MIT
+
+pragma solidity =0.8.25;
+
+import "./halmos-cheatcodes/src/SymTest.sol";
+import {Test, console} from "forge-std/Test.sol";
+
+contract GlobalStorage is SymTest {
+    // uint256->address mapping to have an ability to iterate over addresses
+    mapping (uint256 => address) addresses;
+    mapping (address => string) names_by_addr;
+
+    uint256 addresses_list_size = 0;
+
+    // Addresses and names information is stored using this setter
+    function add_addr_name_pair (address addr, string memory name) public {
+        addresses[addresses_list_size] = addr;
+        addresses_list_size++;
+        names_by_addr[addr] = name;
+    }
+
+    /*
+    ** It is expected to receive a symbolic address as a parameter
+    ** This function should return some concrete address and corresponding data.
+    ** In the case of symbolic execution, the brute force over addresses
+    ** is happening here!
+    */
+    function get_concrete_from_symbolic (address /*symbolic*/ addr) public view 
+                                        returns (address ret, bytes memory data) 
+    {
+        for (uint256 i = 0; i < addresses_list_size; i++) {
+            if (addresses[i] == addr) {
+                string memory name = names_by_addr[addr];
+                return (addresses[i], svm.createCalldata(name));
+            }
+        }
+        revert(); // Ignore cases when addr is not some concrete known address
+    }
+}
+```
+
+We will not dwell on the implementation details of this contract. I will only say that this is a contract in which you can store address->contract name pairs. And also with its help you can conveniently brute force addresses symbolically. It is easier to show how to use it in practice. First, let's prepare Global Storage in Truster_Halmos.t.sol:
```

Comment:
nit: can we rename the file TrusterHalmos.t.sol? let's drop the underscore, it's cleaner